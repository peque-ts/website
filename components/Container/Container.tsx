import clsx from 'clsx';
import React from 'react';

interface Props {
  className?: string;
}

export const Container: React.FC<Props> = ({ children, className }) => (
  <div className={clsx('container mx-auto px-4', className)}>{children}</div>
);
