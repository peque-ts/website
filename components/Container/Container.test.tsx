import { render } from '@testing-library/react';

import { Container } from './Container';

describe('Container', () => {
  it('should render', () => {
    const { container, asFragment } = render(<Container className="pizza" />);

    expect(container.children[0]).toHaveClass('container mx-auto px-4 pizza');
    expect(asFragment()).toMatchSnapshot();
  });
});
