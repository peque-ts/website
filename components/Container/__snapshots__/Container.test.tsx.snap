// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render 1`] = `
<DocumentFragment>
  <div
    class="container mx-auto px-4 pizza"
  />
</DocumentFragment>
`;
